File: C/Assignments/Assignment 0/q9.c
//9. Write a C program to input the base and height of a triangle and calculate its area.

 void main()  {
	float h , b ,area ;
	
	printf("enter the height :  ");
	scanf("%f" , &h) ;
	printf("\n enter the base : ");
	scanf("%f" , &b) ;
	
	
	area = 1.0/2 * (h * b) ;
	
	printf("area is %f" , area);


}

File: C to Java/Assignments/Assignment 0/q9.java
import java.util.Scanner;

public class q9 {

    public static void main(String args[]) {

        Scanner sc = new Scanner(System.in);

        	double h , b ,area ;
	
	System.out.printf("enter the height :  ");
	h = sc.nextDouble();

	System.out.printf("\n enter the base : ");
	b = sc.nextDouble();

	
	
	area = 1.0/2 * (h * b) ;
	
	System.out.printf("area is %f" , area);

        }
}